src/target/llvm/codegen_cpu.cc

```diff
@@ -300,6 +296,36 @@ llvm::DIType* CodeGenCPU::GetDebugType(const Type& ty_tir, llvm::Type* ty_llvm)
                                              : nullptr;
     return dbg_info_->di_builder_->createPointerType(pointee_type,
                                                      ty_llvm->getPrimitiveSizeInBits());
+
+  } else if (auto* prim_type = ty_tir.as<PrimTypeNode>()) {
+    DataType dtype = prim_type->dtype;
+    auto [name, dwarf_type] = [&]() -> std::tuple<const char*, llvm::dwarf::TypeKind> {
+      if (dtype.is_bool()) {
+        return {"bool", llvm::dwarf::DW_ATE_boolean};
+      } else if (dtype.is_float()) {
+        return {"float", llvm::dwarf::DW_ATE_float};
+      } else if (dtype.is_int()) {
+        return {"int", llvm::dwarf::DW_ATE_signed};
+      } else if (dtype.is_uint()) {
+        return {"uint", llvm::dwarf::DW_ATE_unsigned};
+      } else {
+        LOG(FATAL) << "No DWARF representation for TIR type " << dtype;
+      }
+    }();
+
+    std::stringstream ss;
+    ss << name;
+
+    if (!dtype.is_bool()) {
+      ss << dtype.bits();
+    }
+    if (dtype.lanes() > 1) {
+      ss << "x" << dtype.lanes();
+    }
```

Comment:
There is already a function that gets the string from `DataType` [here](https://github.com/apache/tvm/blob/51dcafb4abc7dbc3583c1719d3c483ae52d67859/include/tvm/runtime/data_type.h#L316).

Please add `#include <sstream>` for `std::stringstream` if you decide to keep it.

Reply:
Thank you, and updated to use `DLDataType2String`.